<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMController</class>
 <widget class="QDialog" name="TMController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>TMController</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <widget class="QCheckBox" name="noScreenshotCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>260</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>No Screenshot</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="keepLocalCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Keep local copies</string>
       </property>
      </widget>
      <widget class="QPushButton" name="selectLocationPushButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>71</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Select Location:</string>
       </property>
      </widget>
      <widget class="QLabel" name="folderPath">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>folder path</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="continuousCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>161</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Continous mode</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>561</width>
         <height>78</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Interval</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="intervalMinutesSlider">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="intervalMinutesLabel">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>minutes</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="screenshotSettingsOKButton">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>390</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Settings</string>
       </property>
      </widget>
      <widget class="QLabel" name="screenshotSettingsLabel">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>370</y>
         <width>461</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>status</string>
       </property>
      </widget>
      <widget class="QPushButton" name="openFolderPushButton">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>70</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Open Folder</string>
       </property>
      </widget>
      <widget class="QComboBox" name="startHourComboBox">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>320</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <property name="currentText">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>320</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Select the hour that start a day:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Account</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>pertime.org Account Info</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>661</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>  Email:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="emailEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>661</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>API Key:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="apikeyEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>661</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Computer Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="computernameEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="confirmButton">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>240</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
      <widget class="QLabel" name="confirmedLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>240</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>✓</string>
       </property>
      </widget>
      <widget class="QLabel" name="isrunningLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>400</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Time-Out running status</string>
       </property>
      </widget>
      <widget class="QPushButton" name="closeButton">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>390</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>20</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Check Update</string>
       </property>
      </widget>
      <widget class="QLabel" name="versionLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>371</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>versionLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
